starter-code/pizza.js

```diff
@@ -1 +1,10 @@
-// Write your Pizza Builder JavaScript in this file.
+'use strict'
+
```

Comment:
You have all your code (and therefore your variables) in the global scope. Be careful with that. Better wrap your code in a function that you will run when de window load (or the document is ready if you use jQuery).